src/css/custom.css

```diff
@@ -54,3 +54,7 @@
   position: relative;
   margin-left: 10px;
 }
+
+.flex {
+  display: flex;
+}
```

Comment:
I wonder how frequently we're going to see CSS changes to accommodate doc layouts 🤔 Is it worth adding a little blurb about how and where to add CSS?

Reply:
I could add a sentence in the doc in this PR about CSS changes, I'll do that with my next push.